Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -245,10 +245,13 @@ static wxString dump_textures_desc =
     wxTRANSLATE("Dump decoded game textures to User/Dump/Textures/<game_id>/.\n\nIf unsure, leave "
                 "this unchecked.");
 static wxString load_hires_textures_desc = wxTRANSLATE(
-    "Load custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this unchecked.");
-static wxString cache_hires_textures_desc =
-    wxTRANSLATE("Cache custom textures to system RAM on startup.\nThis can require exponentially "
-                "more RAM but fixes possible stuttering.\n\nIf unsure, leave this unchecked.");
+    "Load custom textures from User/Load/Textures/<game_id>/.\nDolphin now supports "
+    "DXTn compresssed textures natively, reducing the amount of required RAM by more than 5 times."
+    "\n\nIf unsure, leave this unchecked.");
+static wxString cache_hires_textures_desc = wxTRANSLATE(
+    "Prefetch custom textures to system RAM on startup.\nThis can require exponentially "
+    "more RAM for non-native texture formats such as PNG, but fixes possible stuttering."
```

Comment:
This requires additional RAM for DDS too (though not as much additional RAM as in the past). Please undo this change.